src/Test/Desktop/OpenRiaServices.Common.DomainServices.Test/Baselines/Default/EF/Catalog_EFCore.g.cs

```diff
@@ -0,0 +1,2697 @@
+            set
+            {
+                if ((this._modifiedDate != value))
+                {
+                    this.OnModifiedDateChanging(value);
+                    this.RaiseDataMemberChanging("ModifiedDate");
+                    this.ValidateProperty("ModifiedDate", value);
+                    this._modifiedDate = value;
+                    this.RaiseDataMemberChanged("ModifiedDate");
+                    this.OnModifiedDateChanged();
+                }
+            }
+        }
+        
+        /// <summary>
+        /// Gets or sets the 'Name' value.
+        /// </summary>
+        [DataMember()]
+        [Required()]
+        [StringLength(50)]
+        public string Name
+        {
+            get
+            {
+                return this._name;
+            }
+            set
+            {
+                if ((this._name != value))
+                {
+                    this.OnNameChanging(value);
+                    this.RaiseDataMemberChanging("Name");
+                    this.ValidateProperty("Name", value);
+                    this._name = value;
+                    this.RaiseDataMemberChanged("Name");
+                    this.OnNameChanged();
+                }
+            }
+        }
+        
+        /// <summary>
+        /// Gets or sets the 'ProductID' value.
+        /// </summary>
+        [DataMember()]
+        [Editable(false, AllowInitialValue=true)]
+        [Key()]
+        [RoundtripOriginal()]
+        public int ProductID
+        {
+            get
+            {
+                return this._productID;
+            }
+            set
+            {
+                if ((this._productID != value))
+                {
+                    this.OnProductIDChanging(value);
+                    this.ValidateProperty("ProductID", value);
+                    this._productID = value;
+                    this.RaisePropertyChanged("ProductID");
+                    this.OnProductIDChanged();
+                }
+            }
+        }
+        
+        /// <summary>
+        /// Gets or sets the 'ProductLine' value.
+        /// </summary>
+        [DataMember()]
+        [StringLength(2)]
+        public string ProductLine
+        {
+            get
+            {
+                return this._productLine;
+            }
+            set
+            {
+                if ((this._productLine != value))
+                {
+                    this.OnProductLineChanging(value);
+                    this.RaiseDataMemberChanging("ProductLine");
+                    this.ValidateProperty("ProductLine", value);
+                    this._productLine = value;
+                    this.RaiseDataMemberChanged("ProductLine");
+                    this.OnProductLineChanged();
+                }
+            }
+        }
+        
+        /// <summary>
+        /// Gets or sets the 'ProductModelID' value.
+        /// </summary>
+        [DataMember()]
+        public Nullable<int> ProductModelID
```

Comment:
Catalog_EFDbCtx.g.cs has "RoundtripOriginal" attribute on this property, but I don't se it explicitly added on the DbContextModel ..

We should investigate why, if it is related to Foreign keys, relations or something else.
There are a few similar differences which I don't understand at a quick glance.

Reply:
I am fine with merging this anyway as long as these "todo's/investigations/remaining work" needed are clearly documented   (using issues or README/TODO for the project) to make it clear what is done or not.

Reply:
other properties `SizeUnitMeasureCode, WeightUnitMeasureCode, EmployeeID, ShipMethodID, VendorID`,

Reply:
Investigated a some of these properties.

They are not flagged with RoundTripOriginal since they are not considered as FK properties and thus will not have the RoundTripAttribute added. This is correct since we for example have decided to exclude both `Vendor` and `UnitMeasure` in the EF Core version of the adventure works model which only contains a subset of the original one. Hence to get the roundtrip attribute we need to extend the model with these entities and reference them.

It still seems to work though, since it was generated correct for for example PurchaseOrderID where both properties was contained in the model.

Added this as a task and added a point in the issue.